src/riot/Components/ServiceWorker.riot.html

```diff
@@ -2,17 +2,23 @@
     <script>
         import { ON_ADD_TO_HOME_SCREEN } from "js/Events";
 
+        const EVERY_FOUR_HOURS = 1000 * 60 * 60 * 4;
```

Comment:
```suggestion
        const SERVICE_WORKER_POLL_INTERVAL = 1000 * 60 * 60 * 4; // every four hours
```
otherwise we have to change the variable name, if we change this :)

Reply:
Good idea. I named it `SW_UPDATE_INTERVAL`.